Error running docker build

(12/15) Installing gcc (9.3.0-r0)
(13/15) Installing musl-dev (1.1.24-r2)
(14/15) Installing go (1.13.13-r0)
(15/15) Installing make (4.2.1-r2)
Executing busybox-1.31.1-r9.trigger
OK: 428 MiB in 29 packages
cd frontend && npm install && npm run build && cd -
/bin/sh: npm: not found
make: *** [Makefile:27: frontend/public/index.html] Error 127
The command '/bin/sh -c apk update &&     apk add -u $build_deps $runtime_deps &&     adduser -D -H -h / -u $user_id tty-server &&     cd /go/src/github.com/elisescu/tty-server &&     GOPATH=/go dep ensure &&     GOPATH=/go make all &&     cp tty-server /usr/bin/ &&     rm -r /go &&     apk del $build_deps' returned a non-zero code: 2

I think Dockerfile file needs to update to build a docker image.
I created an empty file under "frontend/public/index.html", later it asked me to install git, after adding git by "apk add git", I held with this error.

Executing busybox-1.31.1-r9.trigger
OK: 444 MiB in 35 packages
go get github.com/go-bindata/go-bindata/...
# github.com/go-bindata/go-bindata/go-bindata
loadinternal: cannot find runtime/cgo
go-bindata --prefix frontend/public/ -o assets_bundle.go frontend/public/*
/bin/sh: go-bindata: not found
make: *** [Makefile:19: assets_bundle.go] Error 127

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,8 +1,8 @@
-FROM alpine:3.11
+FROM alpine:3.12
 
 ENV URL=http://localhost:5000
 
-ARG build_deps="go make dep"
+ARG build_deps="go make dep git npm"
 ARG runtime_deps="dumb-init"
 ARG user_id=1000
 
@@ -11,9 +11,10 @@ COPY . /go/src/github.com/elisescu/tty-server
 RUN apk update && \
     apk add -u $build_deps $runtime_deps && \
     adduser -D -H -h / -u $user_id tty-server && \
-    cd /go/src/github.com/elisescu/tty-server && \
+    cd /go/src/github.com/elisescu/tty-server/frontend && \
+    npm install && npm build && cd .. && \
     GOPATH=/go dep ensure && \
-    GOPATH=/go make all && \
+    GOPATH=/go PATH=$GOPATH/bin:$PATH make all && \
     cp tty-server /usr/bin/ && \
     rm -r /go && \
     apk del $build_deps

